.mergify.yml

```diff
@@ -0,0 +1,60 @@
+pull_request_rules:
+  - name: backport patches to cryostat-v2.4 branch
+    conditions:
+      - base=main
+      - label=backport
+    actions:
+      backport:
+        branches:
+          - cryostat-v2.4
+        assignees:
+          - "{{ author }}"
```

Comment:
This backport configuration will need to change. Right now it's meaningless to talk about backports for 3.0 since there is only a main branch and no release branches. We might try to predict that the first release branch would be `cryostat-v3.0` for this repo, or just remove/comment out this configuration until the time comes that we're really gearing up for the first 3.0 release and 3.1 branchpoint.

Reply:
yeah I was meaning to ask you about this. Makes sense now